molecule/default/verify.yml

```
@@ -28,3 +28,33 @@
         - gitlab_edition is defined
         - gitlab_version is defined
         - gitlab_release is defined
+
+    - name: "Check GitLab Health endpoint"
+      ansible.builtin.uri:
+        url: "http://localhost/-/health"
+      register: "health_check"
+      failed_when: "'OK' not in health_check.msg or health_check.status!=200"
```

Do you think this more explicit check of the exit code could be beneficial although it is of course implicitly checked by the status code?
 
```suggestion
      failed_when: "health_check.rc != 0 or 'OK' not in health_check.msg or health_check.status!=200"
```